### PR TITLE
Add source_path and test_path parameters

### LINKED ISSUE #1: Initial utPLSQL client features
The client will be developed along with the Java API, so we can identify features needed on the api. Initially the client will support same features as the ut_run.sql script:
- Run tests using the API.
- Show tests results and progress using reporters.
- Accept a path list of tests to be executed.
- Save results to file.

Commands syntax example:
utplsql run user/password@database [params]

### LINKED ISSUE #5: Add Support for source_path and test_path Parameters
Add Support for source_path and test_path parameters like in the sql-cli.

### PR DESCRIPTION
**Pull request by viniciusam**
Related to #17 
Closes #5
Closes #1

### CODE REVIEW

File: src/main/java/io/github/utplsql/cli/RunCommand.java [hunk 1]
```diff
@@ -49,6 +50,12 @@
             description = "enables printing of test results in colors as defined by ANSICONSOLE standards")
     private boolean colorConsole = false;
 
+    @Parameter(names = {"-source_path"}, description = "path to project source files")
+    private String sourcePath;
+
+    @Parameter(names = {"-test_path"}, description = "path to project source files")
+    private String testPath;
```
Comment (jgebal) [inline src/main/java/io/github/utplsql/cli/RunCommand.java]: Wrong description here

File: src/main/java/io/github/utplsql/cli/RunCommand.java [hunk 2]
```diff
@@ -91,6 +98,19 @@ public void run() throws Exception {
         final List<String> testPaths = getTestPaths();
         final List<Reporter> reporterList = new ArrayList<>();
 
+        final File baseDir = new File("").getAbsoluteFile();
```
Comment (jgebal) [inline src/main/java/io/github/utplsql/cli/RunCommand.java]: Is this using current dir(from where cli was invoked)? If so great!

Comment (viniciusam) [inline src/main/java/io/github/utplsql/cli/RunCommand.java]: Yes!

File: src/main/java/io/github/utplsql/cli/RunCommand.java (hunk 1 above)
Comment (viniciusam) [inline src/main/java/io/github/utplsql/cli/RunCommand.java]: Will fix.

**Pull request merged**